ava-test/echo.js

```diff
@@ -129,3 +129,9 @@ test.cb('piping to a file with -n', t => {
     t.end();
   });
 });
+
+test('stderr with unrecognized options is null', t => {
+  const ret = shell.echo('-asdf');
```

Comment:
@freitagbr will this output to the console? If so, let's at least add a TODO to mute console output